src/util.ts

```diff
@@ -1,23 +1,45 @@
 import _ from 'lodash'
 import { Config } from './handler'
 
+interface ChooseUsersResponse {
+  teams: string[]
+  users: string[]
+}
+
 export function chooseUsers(
   candidates: string[],
   desiredNumber: number,
   filterUser: string = ''
-): string[] {
-  const filteredCandidates = candidates.filter(
-    (reviewer: string): boolean => {
-      return reviewer !== filterUser
+): ChooseUsersResponse {
+  const { teams, users } = candidates.reduce(
+    (acc: ChooseUsersResponse, reviewer: string): ChooseUsersResponse => {
+      const isTeam = reviewer.startsWith('/')
+      if (isTeam) {
+        const team = reviewer.slice(1)
```

Comment:
Hey @kentaro-m, I agree. I was coming from the understanding the Github API accepts only team slugs after the slash, so the `org` part would have to be stripped anyway. Hence it will be only useful for people looking at the config file and might get potentially out of sync with reality should the Github organisation name change. However, I don't see any problem with adding support for this syntax! Will do!